src/mocks/freedom-mocks.ts

```diff
@@ -20,6 +20,11 @@ class MockCore {
 
   public getId = () => { return ['useless']; }
 
+  public getLogger = (tag) => {
```

Comment:
This is the freedom core right? The freedom core does have a `getLogger` method and it is used.